components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/services/InputValidationManager.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
```

Comment:
Incorrect license header. check other places as well
```suggestion
 * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```